app/src/main/java/org/jellyfin/androidtv/playback/MediaManager.java

```diff
@@ -227,8 +227,12 @@ public void onPlayerError(ExoPlaybackException error) {
                 mLibVLC = new LibVLC(TvApp.getApplication(), options);
 
                 mVlcPlayer = new org.videolan.libvlc.MediaPlayer(mLibVLC);
-                mVlcPlayer.setAudioOutput(Utils.downMixAudio() ? "opensles_android" : "android_audiotrack");
-                mVlcPlayer.setAudioOutputDevice("hdmi");
+                if(!Utils.downMixAudio()) {
+                    mVlcPlayer.setAudioDigitalOutputEnabled(true);
+                } else {
+                    mVlcPlayer.setAudioOutput("opensles_android");
```

Comment:
I'm not sure if it's necessary or not, but it seems to be inconsistent on whether `mVlcPlayer.setAudioOutputDevice("hdmi");` is called or not. Maybe it should be still be called here?

Reply:
Should be addressed